apps/settings/src/views/Users.vue

```diff
@@ -81,17 +81,14 @@
 			<template #footer>
 				<NcAppNavigationSettings>
```

Comment:
```suggestion
				<NcAppNavigationSettings :exclude-click-outside-selectors=".vs__dropdown-menu">
```

To fix closing if you click the dropdown menu  you will have to add it to the exclude list of the *click outside* trigger.

Reply:
unfortunately i can't see this prop in current Nc Vue version and can't build server with this prop too ;(

Reply:
Its in the docs: https://nextcloud-vue-components.netlify.app/#/Components/App%20containers/NcAppNavigation?id=ncappnavigationsettings

But currently broken (fix is not released yet), to work we have to wait for 7.11.3.

Reply:
> But currently broken (fix is not released yet), to work we have to wait for 7.11.3.

It is merged and should now work if you rebase the PR :)